recipes/jungle/all/patches/0001-cmake-alterations.patch

```diff
@@ -0,0 +1,82 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -13,9 +16,10 @@ endif ()
+ message(STATUS "Build type: " ${CMAKE_BUILD_TYPE})
+ 
+ 
+-set(ROOT_SRC ${PROJECT_SOURCE_DIR}/src)
+-set(CMAKE_DIR ${PROJECT_SOURCE_DIR}/cmake)
+-set(FDB_LIB_DIR ${PROJECT_SOURCE_DIR}/third_party/forestdb/build)
++set(TOP_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
++set(ROOT_SRC ${TOP_SOURCE_DIR}/src)
++set(CMAKE_DIR ${TOP_SOURCE_DIR}/cmake)
++set(FDB_LIB_DIR ${TOP_SOURCE_DIR}/third_party/forestdb/build)
+ 
+ if (CODE_COVERAGE GREATER 0)
+     set(CMAKE_BUILD_TYPE "Debug")
+@@ -45,10 +49,10 @@ set(LIBDL dl)
+ 
+ # Includes
+ include_directories(BEFORE ./)
+-include_directories(BEFORE ${PROJECT_SOURCE_DIR}/src)
```

Comment:
Changes here seem a little much, https://cmake.org/cmake/help/latest/variable/PROJECT_SOURCE_DIR.html

> source directory of the last call to the [project()](https://cmake.org/cmake/help/latest/command/project.html#command:project) command made in

I am surprised this is required 🤔 

***

Can you confirm this was needed 🙏

Reply:
You are correct, vestigial patch it seems.